server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -1537,6 +1541,7 @@ public void getMessageListHasKeywordShouldIgnoreRecent() throws Exception {
     }
 
     @Test
+    @SuppressWarnings("unchecked")
```

Comment:
there is a better way to fix these, see and cherry pick 8b37d572f2b4462f2c61f2a4b8d0413d05810a66